Sources/StreamChatUI/Utils/UIViewController+Extensions.swift

```diff
@@ -19,4 +19,90 @@ extension UIViewController {
         view.removeFromSuperview()
         removeFromParent()
     }
+
+    var isUIHostingController: Bool {
+        String(describing: self).contains("UIHostingController")
+    }
+
+    /// Helper method to correctly setup navigation of parent.
+    /// Used when view is wrapped using `UIHostingController` and used in SwiftUI
+    /// so all properties of `navigationItem` are propagated correctly.
+    @available(iOS 13.0, *)
+    func setupParentNavigation(parent: UIViewController) {
```

Comment:
Let's add a test for this func. The logic here is quite tricky.

Reply:
Tests added.